### ISSUE TITLE
Hi Everyone, Where to go moving forward?

### ISSUE THREAD

**Issue by Tweekism** [iamcco/markdown-preview.nvim#688]
**EDIT:** Change of plans, I'll be heading over [here](https://github.com/jannis-baum/vivify). Deets in the update reply below...

Hi Team,

It seems possible that iamcco may no longer be maintaining this project. This is a fantastic extension and it would be a shame to see it go.

I've reached out to iamcco via email to see if he is still interested in maintaining the project or if he'd be willing to open up this repo to others. We'll have to wait and see if he responds and if he does I'll update here.

In the meantime I'd be happy to accept pull requests at my fork [Tweekism/markdown-preview.nvim](https://github.com/Tweekism/markdown-preview.nvim) until iamcco returns.

Iamcco, if you see this reach out bud, we miss you :)

**Comment by Tweekism** [comment]
Update: So I've not heard back from iamcco unfortunately.

However in my digging I did run into a couple of previous users by looking through old pull requests. They have spun off their own replacement project that is, I guess, inspired by this one but is a rewrite from scratch. It is shaping up quite nicely, already has the main features of this plugin and is being actively worked on. 

So in short I'll be heading over there.

If anyone else is interested: https://github.com/jannis-baum/vivify

**Comment by M-Arnone** [comment]
I'll check this, thanks!

**Comment by pidgeon777** [comment]
> However in my digging I did run into a couple of previous users by looking through old pull requests. They have spun off their own replacement project that is, I guess, inspired by this one but is a rewrite from scratch. It is shaping up quite nicely, already has the main features of this plugin and is being actively worked on.

Hello, is there any link available?

**Comment by Tweekism** [comment]
Hey mate, yeah it's this one: https://github.com/jannis-baum/Vivify

It operates a little differently in that it can also run standalone without neovim. As a consequence there's a thing you have to install, but since it's fairly new they don't have packages for every distro yet.

Anyway, if you need help I'm watching the discussions over there, so just ask.

I'm also still considering reviving this, maybe, possibly, so watch this space I guess. 😅

**Comment by Tweekism** [comment]
Here is the original issue discussing the new project of thats what you are looking for, its the last couple of posts in here: https://github.com/iamcco/markdown-preview.nvim/issues/58

**Comment by pidgeon777** [comment]
Thank you @Tweekism.

I had a look at the `Vivify` project, seems good but it doesn't seem to support Windows, unfortunately...

**Comment by Tweekism** [comment]
Riiiight.... Forgot about that.

A Windows version is feasible, but they need someone who uses Windows and is willing to help develop the port.

**Comment by Lazerbeak12345** [comment]
https://github.com/toppair/peek.nvim seems like a decent possiblity too, if you prefer it to be a part of your ide, and you're ok with having deno installed. Doesn't support a lot of the same file types.

**Comment by icedwater** [comment]
> Riiiight.... Forgot about that.
> 
> A Windows version is feasible, but they need someone who uses Windows and is willing to help develop the port.

Hoping for a hero. I'd wade in, but I haven't looked at the code enough to know if I can port it, plus my Windows machine is only used during office hours.

**Comment by Tweekism** [comment]
> Hoping for a hero. I'd wade in, but I haven't looked at the code enough to know if I can port it, plus my Windows machine is only used during office hours.

Yeah I was in kinda the same boat, so I did the only logical thing... change my work PC over to Linux as well. 🙃

We have had a few people express interest in a Windows version. @icedwater are you using vim in Windows native or WSL?

**Comment by icedwater** [comment]
Both vim in git bash (9.1.1806) as well as via chocolatey (9.1.1904).

**Comment by Tweekism** [comment]
Thanks, I'll let the Vivify people know, might help with planning which direction to focus effort on.

EDIT: oh nevermind, i remember you already posted in the windows thread over there lol.
